### PR TITLE
Rename lists to views

### PR DESCRIPTION
**Pull request by kaulfield23**
## Description
This PR fixes a bug where it breaks localization in view

## Screenshots
image

## Changes

* Changes `lists` to `views`

## Related issues
Undocumented

### CODE REVIEW

**Review by richardolsson — CHANGES_REQUESTED** [review]
The changes to the `makeMessages()` prefix looks good to me, but I don't understand why we would need to change the test. They should be unrelated, and to me the test looked right before (using "list"). If that test failed, I believe that it's either flaky (retry it) or the code it's testing is wrong (fix it).

File: integrationTesting/tests/organize/views/detail/create.spec.ts [hunk 1]
```diff
@@ -78,7 +78,7 @@ test.describe('View detail page', () => {
     expect(columnPostLogs).toHaveLength(2);
 
     // Expect that correctly localised strings sent when posting
-    expect(viewPostLogs[0].data?.title).toEqual('New list');
+    expect(viewPostLogs[0].data?.title).toEqual('New View');
```
[the remark this answers is not in the document]
Comment (kaulfield23) [reply integrationTesting/tests/organize/views/detail/create.spec.ts:81]: The test failed there and result said it was expecing to have `New list` but the result was `New View` so I fixed it to `New View` @richardolsson

[the remark this answers is not in the document]
Comment (kaulfield23) [reply integrationTesting/tests/organize/views/detail/create.spec.ts:81]: image

**Review by richardolsson — APPROVED** [review]
I'm not sure why I didn't already review this, because it looks fine and simple!

**Pull request merged**